components/kyma-environment-broker/internal/process/staged_manager.go

```diff
@@ -136,9 +136,11 @@ func (m *StagedManager) Execute(operationID string) (time.Duration, error) {
 				logStep.Debugf("Skipping")
 				continue
 			}
+			events.Infof(operation.InstanceID, operation.ID, "processing step: %v", step.Name())
```

Comment:
@piotrmiskiewicz recommended to try putting helper method to `Operation` object, that way it would be even shorter
```go
operation.InfoEvent(message)
```

Reply:
hmm, turns out this is a bit harder due to architecture of KEB packages and import cycle
```
package command-line-arguments
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/common/hyperscaler
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/driver/postsql/events
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/dbmodel
        imports github.com/kyma-project/control-plane/components/kyma-environment-broker/internal: import cycle not allowed
```
db packages imports `Operation` from `internal` package and as a result, it's non-trivial to import db packages in the `internal` package. I'd like to avoid refactoring here so skipping this suggestion for now